ArduPlane/Parameters.cpp

```diff
@@ -522,13 +522,30 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Standard
     GSCALAR(fs_timeout_long,        "FS_LONG_TIMEOUT", 5),
 
+    // @Param: FS_GCS_TIMEOUT
+    // @DisplayName: GCS failsafe timeout
+    // @Description: The time in seconds that a GCS failsafe condition has to persist. This defaults to 0 seconds which means "disabled".  
+    // @Units: s
+    // @Range: 3 2147483647
```

Comment:
We can lower the maximum range limit. Users can always set the value to whatever they want, but the range is used to generate slider objects normally in UI code, and this is a crazy scale for the slider.